learning/features/audio/audio_buses.rst

```diff
@@ -159,12 +159,12 @@ It will be sufficient to note that you can make a Darth Vader voice with it, or
 PitchShift
 ~~~~~~~~~~
 
-This effect allows modulating pitch separatedly from tempo. All frequencies can go and up and down with transients kept as bet as possible. It's amazing for making funny voices!
+This effect allows modulating pitch separatedly from tempo. All frequencies can go and up and down with transients kept as well as possible. It's amazing for making funny voices!
```

Comment:
Not sure about this one.

Reply:
"separatedly" is not a word. :P

"It's amazing..." is much too informal and opinionated.  How about:

This effect allows for modulating pitch independently of tempo. All frequencies can be increased/decreased with transients kept to a minimum.  Can be used for effects such as voice modulation.